README.md

```diff
@@ -32,40 +32,34 @@ locking an element.
 If an element is locked for display, it means that any DOM updates to its
 subtree are not rendered immediately. Instead, when the lock is committed, the
 user-agent processed rendering updates co-operatively, yielding periodically to
-allow script or user interactions to happen. 
+allow script or user interactions to happen.
 
-The visual content of an element which is locked for display does not change.
-One option is that the visual output is fixed to whatever content was present at
-the time the lock was acquired, and remains this way until the lock is released
-and all associated promises are resolved. Another possiblity is effectively
-making the locked subtree `visibility: hidden`, meaning that we don't display
-any content while work is completed.
-
-There are other possible options here, such as displaying developer specified
-content. The details of this are up for discussion, but the consistent result is
-that the visual representation of the modified DOM is not necessarily rendered
-to screen, affording the user-agent the opportunity to break the work up into
-smaller pieces.
+The visual content of an element's subtree which is locked for display does not
+change. Specifically, if the element already existed in the DOM, then the
```

Comment:
Add a sentence saying explicitly that the root element is not locked in the case of an element already in the DOM,
so updates to it are not cached.